WordPress/Classes/ViewRelated/Blog/Site Picker/SiteSwitcherView.swift

```diff
@@ -0,0 +1,141 @@
+import SwiftUI
+import DesignSystem
+
+struct SiteSwitcherView: View {
+    @State private var isEditing: Bool = false
+    private let selectionCallback: ((NSNumber) -> Void)
+    private let addSiteCallback: (() -> Void)
+    private let blogListViewModel = BlogListViewModel()
```

Comment:
Nit: @alpavanoglu This property is not used. Perhaps you wanted to inject it in `BlogListView`?